tests/unit/test_pynblint.py

```diff
@@ -8,28 +8,28 @@
 
 @pytest.fixture(scope="module")
 def notebooks():
-    nb1 = Notebook(Path("../fixtures") / Path("EmptyNotebook.ipynb"))
-    nb2 = Notebook(Path("../fixtures") / Path("FullNotebook.ipynb"))
-    nb3 = Notebook(Path("../fixtures") / Path("JustMarkdown.ipynb"))
-    nb4 = Notebook(Path("../fixtures") / Path("DefectedNotebook.ipynb"))
+    nb1 = Notebook(Path("../fixtures", Path("FullNotebook2.ipynb")))
+    nb2 = Notebook(Path("../fixtures", Path("FullNotebook.ipynb")))
+    nb3 = Notebook(Path("../fixtures", Path("JustMarkdown.ipynb")))
+    nb4 = Notebook(Path("../fixtures", Path("DefectiveNotebook.ipynb")))
```

Comment:
Actually, I meant something like this, so that we call `Path()` only once. 

Make sure it works on your operating system too, please. If not, leave it as it is.

```suggestion
    nb1 = Notebook(Path("../fixtures", "FullNotebook2.ipynb"))
    nb2 = Notebook(Path("../fixtures", "FullNotebook.ipynb"))
    nb3 = Notebook(Path("../fixtures", "JustMarkdown.ipynb"))
    nb4 = Notebook(Path("../fixtures", "DefectiveNotebook.ipynb"))
```

Reply:
yes it works, I already tried previously, just forgot to push the changes after the last commit you reviewed